lib/base/scriptframe.cpp

```diff
@@ -51,7 +55,7 @@ INITIALIZE_ONCE_WITH_PRIORITY([]() {
 
 	Namespace::Ptr internalNS = new Namespace(l_InternalNSBehavior);
 	globalNS->SetAttribute("Internal", std::make_shared<ConstEmbeddedNamespaceValue>(internalNS));
-}, 50);
+}, 1000); /* This. */
```

Comment:
What's the reason for this comment?

Reply:
The comment above which explains the priority. I fear that this is changed or overseen in the future.